Tutorials/NetKet_Tools/custom_graph.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Demonstrates the use of the pynetket to calculate ground state of simple
+Ising1D model.
+"""
+
+from __future__ import print_function
+import json
+import networkx as nx
+import netket_tools as nkt
+
+pars = {}
+
+# defining a custom graph
+# here we use networkx to generate a star graph
+# and pass its edges list to NetKet
+pars['Graph'] = nkt.graph.from_networkx(nx.star_graph(10))
```

Comment:
This is already great, but maybe even better if we could do 
```python
nkt.graph(nx.star_graph(10))
````

I mean, we could define a graph constructor that automatically calls the corresponding converter if the argument is a networkx, or igraph?

Reply:
I've added a new function that takes care of figuring out what type of graph it is automatically. Usage looks like this: 

```python
pars['Graph'] = nkt.graph.from_graph(nx.star_graph(10))
```

Reply:
Alright, but what about a constructor? :) The beauty of short function names...

Reply:
Are you looking for something exactly like your first comment? Like this:

```python
pars['Graph'] = nkt.graph(nx.star_graph(10))
```

Reply:
Yes!! I think this would be great, it'd be more compact and easier to remember...